### PR TITLE
Detail-View updating text colors and sizes

### LINKED ISSUE #245: [Detail View Subtitle] Insufficient Color Contrast
# Issue Description

Issue: the subtitle gray text does not pass the accessibility WCAG AA color contrast ratio: 

image

image
contrast between background (#ffffff) and Subtext text color (#BCBFC0).
## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Current Behavior

The subtitle gray color of the detail view does not pass the WCAG AA contrast ratio.

## Steps to Reproduce

1. 
2.
3.
4.

## Environment

* Component Version:
* Browser Name and Version: Chrome 
* Operating System and version (desktop or mobile): OS desktop

### LINKED ISSUE #257: Detail View minor visual fixes
# Issue Description
Minor updates to fix UI inconsistencies from TRVM uplifts to new Terra-UI/React 

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## UI Design Direction
((Double checking the fixed values and will add here))

### PR DESCRIPTION
**Pull request by neilpfeiffer**
### Summary
Updating the text colors and sizes to better match previous TRVM builds
and correct accessibility color-contrast problems.

Resolves #257, Resolves #245 

### Deployment Link
https://terra-clinical-deployed-pr-297.herokuapp.com/static/#/site/components/clinical-site/detail-view/index

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
It may be good to get #286 in to ensure accessibly passes as expected. This PR would add the accessibility tests.

**Comment by PaulSolDev** [comment]
I noticed some visual inconsistencies across browsers on the Divided Detail View when it comes to the "CBC Results" section in terms of text wrapping and line spacing. Browsers in image below from Left to Right (Edge, IE10, IE11, Firefox, Chrome)

image

**Comment by neilpfeiffer** [comment]
Looking into layout issues and will generate wdio tests

File: packages/terra-clinical-detail-view/src/DetailList.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 :local {
   .title {
-    font-size: 1.285rem;
-    padding-top: 10px;
+    font-size: 1.2857142857rem;
+    line-height: 1.4285714286rem;
```
Comment (emilyrohrbough) [inline packages/terra-clinical-detail-view/src/DetailList.scss]: Line-height should be unitless

**Comment by emilyrohrbough** [comment]
Looks like there are a few screenshots that need updated with these changes.

File: packages/terra-clinical-detail-view/src/DetailList.scss (hunk 1 above)
Comment (neilpfeiffer) [reply packages/terra-clinical-detail-view/src/DetailList.scss]: good catch! thanks

**Comment by neilpfeiffer** [comment]
@ryanthemanuel - I completed updating the values to the new sizes & colors and regenerated reference screenshots to match new visuals and have everything passing. This is ready-to-go after Functional Review and your +1.

@PaulSolDev - this is now ready to verify - test page is here:
https://terra-clinical-deployed-pr-297.herokuapp.com/static/#/site/components/clinical-site/detail-view/index

I found the problems you were seeing above in https://github.com/cerner/terra-clinical/pull/297#issuecomment-370901696 were due to the example trying to render a graph image that wasn't a valid url. I updated the example to be simpler until we have a better sample of detail view content.

\
What's changed: _details are listed in the issue description -->_ #257

**Comment by PaulSolDev** [comment]
Functional review complete. Previously identified issues have been addressed and no additional issues found.

**Pull request merged**